### PR TITLE
Feat: Lt-2637/2840 document content

### PR DESCRIPTION
**Pull request by Mark-Spiers-kainos**

### CODE REVIEW

File: letter_templates/context_generator.py [hunk 1]
```diff
@@ -397,6 +511,10 @@ def _get_good_on_application_context(good_on_application, advice=None):
         good_context["item_type"] = good_on_application.item_type
         good_context["other_item_type"] = good_on_application.other_item_type
 
+    # Potential missing scope
+    #
+    # pvgrading details?
```
Comment (efahy) [inline letter_templates/context_generator.py:516]: Don't forget to remove

File: letter_templates/custom_tags.py [hunk 2]
```diff
@@ -52,3 +56,43 @@ def remove_underscores(value):
     Removes the underscores from a given string
     """
     return value.replace("_", " ").title()
+
+
+@register.simple_tag(name="exporter_lcs")
+@mark_safe
+def get_exporter_lite_content_string(value):
+    """
+    Template tag for accessing constants from LITE content library (not for Python use - only HTML)
```
Comment (efahy) [inline letter_templates/custom_tags.py:65]: Can you just say "... from the exporter section of LITE content ..."

File: letter_templates/layouts/application_form.html [hunk 3]
```diff
@@ -218,54 +218,211 @@ <h2 class="govuk-heading-m">Exhibition details</h2>
...
+						<dd class="govuk-summary-list__value">
+							{{ good.description }}
+						</dd>
+					</div>
+					<div class="govuk-summary-list__row">
+						<dt class="govuk-summary-list__key">
+							{% exporter_lcs 'goods.GoodsDetailSummary.CONTROLLED' %}
+						</dt>
+						<dd class="govuk-summary-list__value">
+							{{ good.is_controlled }} - {{ good.control_list_entries|join:", "|default_na }}
```
Comment (efahy) [inline letter_templates/layouts/application_form.html:239]: Can you change this so the yes or no is capitalised like the rest of the yes and nos

Comment (Mark-Spiers-kainos) [reply letter_templates/layouts/application_form.html:239]: Done introduced it in document context by introducing another to_str method for "is_controlled" choices

**Pull request merged**